app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -65,6 +65,12 @@ def batch(from, size)
         hosts.offset(from).limit(size)
       end
 
+      # Will have default in ProxyBatchTriggering if we cant find provider
+      def proxy_batch_size
```

Comment:
I can't find where this gets called

Reply:
I missed that `ProxyBatchTriggering` is in tasks xD and kept wondering why the tests are failing...
https://github.com/theforeman/foreman-tasks/pull/616